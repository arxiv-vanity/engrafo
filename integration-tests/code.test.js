var utils = require("./utils");

test("basic listings render correctly", done => {
  utils.expectBodyToMatchSnapshot("code/basic.tex", done);
});
