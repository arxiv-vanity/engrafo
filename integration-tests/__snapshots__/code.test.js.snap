// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic listings render correctly 1`] = `
<body>
  <div
    class="engrafo-container"
  >
    


    <div
      class="ltx_page_content"
    >
      

      <dt-article>
        <h1
          class=""
        />
        <div
          class="engrafo-metadata"
        >
          <div
            class="engrafo-metadata-custom"
          />
        </div>
        

        <div
          class="ltx_para"
          id="p1"
        >
          

          <div
            class="ltx_listing ltx_lstlisting ltx_listing"
          >
            


            <div
              class="ltx_listingline"
            >
              

              <span
                class="ltx_text ltx_lst_identifier"
              >
                print
              </span>
              (”
              <span
                class="ltx_text ltx_lst_identifier"
              >
                hello
              </span>
              ”)

            </div>
            

            <div
              class="ltx_listingline"
            >
              

              <span
                class="ltx_text ltx_lst_identifier"
              >
                print
              </span>
              (”
              <span
                class="ltx_text ltx_lst_identifier"
              >
                world
              </span>
              ”)

            </div>
            

            <div
              class="ltx_listingline"
            >
              

              <span
                class="ltx_text ltx_lst_identifier"
              >
                if
              </span>
              <span
                class="ltx_text ltx_lst_space"
              >
                 
              </span>
              <span
                class="ltx_text ltx_lst_identifier"
              >
                True
              </span>
              :

            </div>
            

            <div
              class="ltx_listingline"
            >
              

              <span
                class="ltx_text ltx_lst_space"
              >
                  
              </span>
              <span
                class="ltx_text ltx_lst_identifier"
              >
                print
              </span>
              (”
              <span
                class="ltx_text ltx_lst_identifier"
              >
                indents
              </span>
              <span
                class="ltx_text ltx_lst_space"
              >
                 
              </span>
              <span
                class="ltx_text ltx_lst_identifier"
              >
                work
              </span>
              ”)

            </div>
            

          </div>
          

        </div>
        

      </dt-article>
      

      <dt-appendix>
        

        


        <div
          class="l-body"
        />
        

      </dt-appendix>
    </div>
    





    <div
      id="footnote-hover-boxes-container"
    />
  </div>
</body>
`;
